src/oscar/templates/oscar/catalogue/detail.html

```diff
@@ -9,11 +9,11 @@
 {% load purchase_info_tags %}
 
 {% block title %}
-    {{ product.get_title }} | {{ block.super }}
+    {{ product.meta_title|default:product.get_title|default:""|striptags }} | {{ block.super }}
```

Comment:
I suggest that instead of doing this we define a `get_meta_title` method on the product. This method should:

1. Check the `product.meta_title` and return that if it exists.
2. If it's a child product, then check the `parent.meta_title` if it exists.
3. If we still have nothing, return `product.get_title`.

See the implementation of `get_title()` for how to check parent products.

We also don't need `striptags` here - the title does not contain HTML.